Sparkle.xcodeproj/project.pbxproj

```diff
@@ -2816,7 +2816,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export CODE_SIGN_IDENTITY=\"Developer ID Application: Momenta BV\"\n\"$SRCROOT/Configurations/set-agent-signing.sh\"";
+			shellScript = "export CODE_SIGN_IDENTITY=$(EXPANDED_CODE_SIGN_IDENTITY)\n\"$SRCROOT/Configurations/set-agent-signing.sh\"\n";
```

Comment:
Using the expanded identity here avoids problems with people having multiple ambiguous "Mac Developer" identities.

Reply:
Nice solution!